### PR TITLE
fix(argo-workflows): a link in values.yaml for Kubernetes Ingress docs

### PR DESCRIPTION
**Pull request by inamuu**
I couldn’t access the link because it returned a 404 Not Found error. 
Therefore, I updated the link to an existing URL.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Comment by yu-croco** [comment]
Hi @inamuu , thank you for your PR.
Please handle items in your PR description. You checked them but I am afraid you didn't handle properly.

> I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
> I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).

**Comment by inamuu** [comment]
@yu-croco 
Thank you for your reply.
I fixed it.  Please check the commit :pray:

**Review by tico24 — DISMISSED** [review]

**Review by yu-croco — APPROVED** [review]
Thank you for your contribution. LGTM.

File: charts/argo-workflows/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: v3.6.2
+appVersion: v3.6.3
```
Comment (yu-croco) [inline charts/argo-workflows/Chart.yaml:2]: Ah not this one but `version`  in line 6.

Comment (inamuu) [reply charts/argo-workflows/Chart.yaml]: Sorry, I mistaked and fixed it :bow:

**Review by vladlosev — APPROVED** [review]
LGTM

**Review by jmeridth — APPROVED** [review]

**Pull request merged**